packages/appcd-gulp/package.json

```diff
@@ -13,39 +13,40 @@
     "@babel/plugin-transform-async-to-generator": "latest",
     "@babel/polyfill": "latest",
     "@babel/register": "latest",
```

Comment:
Now that babel 7 is released, any objections to replacing latest with ^7.0.0 here?

Reply:
Yes, I will do this! Thanks!